### PR TITLE
Unnecessary global variables.

### PR DESCRIPTION
**Pull request by saby-echo**
module was failing when run with '--use-strict' flag.

### CODE REVIEW

File: lib/range.js [hunk 1]
```diff
@@ -112,8 +112,8 @@ function increaseRangeBitDepth(ranges, radiusBitDepth, bitDepth) {
 
     var bitDiff = bitDepth - radiusBitDepth;
 
-    for (i = 0; i < ranges.length; i++) {
-        range = ranges[i];
+    for (var i = 0; i < ranges.length; i++) {
```
Comment (arjunmehta) [inline lib/range.js]: This is fine.

File: lib/range.js [hunk 2]
```diff
@@ -112,8 +112,8 @@ function increaseRangeBitDepth(ranges, radiusBitDepth, bitDepth) {
...
+        var range = ranges[i];
```
Comment (arjunmehta) [inline lib/range.js]: range should not be redeclared with every iteration. Needs to be pulled out of the loop. I suggest declaring it at the top of the method. View comment on pull request! :)

**Comment by arjunmehta** [comment]
```javascript
function increaseRangeBitDepth(ranges, radiusBitDepth, bitDepth) {

    var bitDiff = bitDepth - radiusBitDepth;
    var range;

    for (var i = 0; i < ranges.length; i++) {
        range = ranges[i];
        range[0] = leftShift(range[0], bitDiff);
        range[1] = leftShift(range[1], bitDiff);
    }
}
```

**Comment by saby-echo** [comment]
I don't think it matters. Look at this..http://stackoverflow.com/questions/3684923/javascript-variables-declare-outside-or-inside-loop. Anyways thanks for fixing the bug. Keep up the good work. Cheers!

-----Original Message-----

**Comment by arjunmehta** [comment]
Thanks!! 

Yeah, it's more convention than anything else. I will probably add an .eslintrc file to the project soon! :)

**Comment by saby-echo** [comment]
Cool

**Comment by arjunmehta** [comment]
Published. Thanks again.

**Pull request merged**